src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -221,13 +222,27 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
     }
   }
 
-  function commitUnmount(current : Fiber) : void {
+  function commitUnmount(current : Fiber, safely : boolean) : void {
+    // Make sure we mark this deletion as completed so that we don't
+    // attempt to perform it again if one of the deletions fails, and
+    // error boundary tries to unmount this subtree again.
```

Comment:
>When we commit, we've already swapped the tree so as far as Fiber is concerned, it is deleted in the "current" state.

Hmm, I think this is something I changed below.
Without changing it, Fiber complained that

>Cannot commit the same tree as before. This is probably a bug related to the return field.

whenever I caught an error during commit phase (e.g. `DidMount`) and tried to get error boundary to re-render the whole thing.

Alternatively I could "roll the pointer back" on error but I didn't want a try catch there.
What should I do?